earth_enterprise/src/fusion/dbmanifest/dbmanifest.cpp

```diff
@@ -962,18 +963,40 @@ void DbManifest::GetMapsJsonFiles(
   LoadLocales(&locale_set);
   // Need to parse the stream url.
   QUrl qurl(stream_url.c_str());
+  int version;
   for (int i = -1; i < static_cast<int>(locale_set.supportedLocales.size());
        ++i) {
     std::string locale = i < 0 ?
         kDefaultLocaleSuffix : std::string(locale_set.supportedLocales[i]);
 
+    // Find the version parameter.
+    std::string versionParam = "v=";
+    std::string query = qurl.query();
+    int vIndex = query.find(versionParam) + 3;
+    version = std::stoi(query.substr(vIndex,1));
+
+    // Validate the version number.
+    if (version != 1 || version != 2) {
+      version = 2;
```

Comment:
The issue wasn't very clear here.  I fixed it to be more clear.  But the default for the first release after this code goes in should be 1 so there is no behavior change to current usages.  Then everyone has one release to convert to the new behavior before we change the default behavior in the next release after the next release.  So I think this should be one.